app/views/users/index.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+<h1>All Users</h1>
+<p><% @users.each do |user| %></p>
+
+<p>User id: <%= user.id %></p>
+<p>User email: <%= user.email %></p>
+
+  <% if user.admin == true %>
```

Comment:
Is this a boolean field in the database? We don't need the `== true` if so. You can just write `if user.admin`